### DISCUSSION TITLE
Vogons.org - FastDoom 800x600 crash with SBEMU

### DISCUSSION THREAD

**Discussion by ZX-Commodore-ST** [viti95/FastDoom#189] [General]
Unfortunately Vogons forums rules doesn't allow me to respond to viti95 message (not enough posts on their forums to send personal messages) so I will do it here. TL:DR I don't think it's a FastDoom error.

It looks like another example of problems that my retro PC has with jemm386.exe. It is a Termtek TK-3770 with 1Ghz VIA CPU & 512 MB RAM & integrated graphics. It is almost the same as this one: https://www.philscomputerlab.com/wortmann-terra--termtek-tk-3772.html

It hangs only when I try it with sbemu and sbemu forces me to use jemm386.exe memory manager which I can't configure properly. To be able to run FastDoom with sbemu I also have to use dos4g.exe instead of dos32a.exe. Higher resolutions doesn't work but default resolution fdoom.exe & fdoom13h.exe works fine

There are no issues when I run FastDoom with PCI sound card Yamaha YMF744. To get a sound on Yamaha I have to load sound drivers but at least I can use different EMS. It's emm386.exe version 2.26: https://fd.lod.bz/repos/current/pkg-html/emm386.html

**Comment by viti95** [comment]
Have you tried the image SBEMU now provides preconfigured (https://github.com/crazii/SBEMU/releases)? I've been using it and pretty much works fine with all games without having to configure much. The only thing you have to do on FastDoom is execute JEMMEX NOVCPI before launching the executable.

I'm thinking the issue in the 800x600 executable comes from the VESA implementantion, some cards are a bit troublesome. What video card does the TK-3770 have?

**Comment by ZX-Commodore-ST** [comment]
I am just after quick test with crazii's image and it doesn't help. Still think it's jemmex being incompatible with my PC. According to HWinfo video card is Via/S3 UniChrome Pro.
